Applications/Classification/README.md

```diff
@@ -1,3 +1,89 @@
-# Classifciation
+# Classification
 
+This example demonstrate the transfer learning with mobilenetv2 which pre-trained for imagenet. "mobilenetv2.tflite" is used for feature extractor which is removed last fully connected layer manually.
 
+After feature extraction, it is used for training fully connencted layer. To take the feature as an input, we add input layer before fully connected layer.
```

Comment:
To take the feature as an input, we add input layer before fully connected layer. -> To take the feature as an input, we add input layer as the first layer?